src/ImageData/FileLoader.cc

```diff
@@ -870,6 +870,23 @@ double FileLoader::CalculateBeamArea() {
     return info.getBeamAreaInPixels(-1, -1, _coord_sys->directionCoordinate());
 }
 
+void FileLoader::ResetBunit() {
```

Comment:
Please rename this to `NormalizeBunit`. Apart from that I think this is good to go (after a re-test).